app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,63 @@
+
+$font-color: #088da5;
+
+body {
+
+  background-color: black;
+
+}
+
+h1,
+h3 {
+  text-align: center;
+  color: $font-color;
+
+}
+
+
+#about {
+
+  border: 3px solid black;
+  text-align: center;
+  padding: 5px;
+  margin: 10px;
+  color: $font-color;
+
+}
+
+#recently-reviewed {
+
+  border: 3px solid black;
+  color: $font-color;
+  margin: 10px;
+  padding: 5px;
+  text-align: center;
+
+}
+
+#new {
+
+  border: 3px solid black;
+  color: $font-color;
+  margin: 10px;
+  padding: 5px;
+  text-align: center;
+
+}
+
+#all-art {
+
+  border: 3px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.